crates/nu-command/tests/commands/def.rs

```diff
@@ -151,15 +151,39 @@ fn def_fails_with_invalid_name() {
 }
 
 #[test]
-fn def_errors_with_specified_list_type() {
+fn def_with_specified_list_type() {
     let actual = nu!(
         cwd: ".", pipeline(
         r#"
         def test-command [ foo: list<any> ] {}
         "#
     ));
 
-    assert!(actual.err.contains("unknown type"));
+    assert!(actual.err.is_empty());
+}
+
+#[test]
+fn def_with_specified_list_type_nested() {
+    let actual = nu!(
+        cwd: ".", pipeline(
+        r#"
+        def test-command [ foo: list<list<any>> ] {}
+        "#
+    ));
+
+    assert!(actual.err.is_empty());
+}
+
+#[test]
+fn def_with_specified_list_type_unterminated() {
+    let actual = nu!(
+        cwd: ".", pipeline(
+        r#"
+        def test-command [ foo: list<any ] {}
+        "#
+    ));
```

Comment:
Might be worth adding a test with a bogus spec like `list<foo>` or invalid separators like `list<foo; bar>`,`list<int, string>`.

Reply:
Thanks!